packages/core/src/components.d.ts

```diff
@@ -169,7 +169,6 @@ export namespace Components {
         "trigger"?: string;
     }
     interface AtomSelect {
-        "color"?: 'primary' | 'secondary' | 'danger';
```

Comment:
perfect. I does not make sense

Reply:
This was left behind by Poc and we never forget to review it hahaha

Reply:
Poc's copying and pasting issues haha